colossalai/tensor/__init__.py

```diff
@@ -1,13 +1,14 @@
-from .spec import ComputePattern, ParallelAction, TensorSpec, ShardPattern
+from .spec import ComputePattern, ParallelAction, TensorSpec
 from .op_wrapper import (
     colo_op_impl,)
 from .colo_tensor import ColoTensor
 from .colo_parameter import ColoParameter
 from .utils import convert_parameter, named_params_with_colotensor
 from ._ops import *
 from .optim.colo_optimizer import ColoOptimizer
+from . import dist_spec
```

Comment:
why not name it as DistSpec?

Reply:
Or DistSignature? to be consistent with our design doc?

Reply:
It's a package not class. I just use package to implement the "namespace" in C++.

Reply:
OK,that's fine

Reply:
I've updated this part in our design doc. I think current implementation is simpler and it's easy to comprehend for users.